### DISCUSSION TITLE
Moving the pet.opendfki.de repository to delph-in

### DISCUSSION THREAD

**Discussion by bkiefer** [delph-in/docs#93] [Team Posts]
Hi,
DFKI tries to get rid of all web pages which are not following some very strict rules, and the opendfki pages are part of that. I feel that the pet repo belongs here, and although i don't know if anybody is still using it, i would be willing to migrate the old content here and take care of all incoming requests.
Please lat me know what you think.

**Comment by fcbond** [comment]
If you are prepared to do that, I think it would be great.

On Tue, 9 Aug 2022 at 10:24, bkiefer ***@***.***> wrote:

> Hi,
> DFKI tries to get rid of all web pages which are not following some very
> strict rules, and the opendfki pages are part of that. I feel that the pet
> repo belongs here, and although i don't know if anybody is still using it,
> i would be willing to migrate the old content here and take care of all
> incoming requests.
> Please lat me know what you think.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/teams/admins/discussions/9>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRVJFEHEPPFT6ZS65VLVYIITPANCNFSM5573VEQA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by bkiefer** [comment]
Could you create a repository for me? It seems i don't have the permissions.

**Comment by olzama** [comment]
I just created a pet repository and added @bkiefer as admin.

**Comment by arademaker** [comment]
It will be great to have Pet in the GitHub! Thank you.
